packages/common/src/PublicNetworks.js

```diff
@@ -43,9 +43,23 @@ const PublicNetworks = {
     daiAddress: "0xbF7A7169562078c96f0eC1A8aFD6aE50f12e5A99",
     wethAddress: "0xd0A1E359811322d97991E03f863a0C30C2cF029C"
   },
+  137: {
+    name: "polygon-matic",
+    etherscan: "https://explorer-mainnet.maticvigil.com/",
+    customTruffleConfig: {
+      confirmations: 2,
+      timeoutBlocks: 200,
+      gasPrice: 1000000000 // 1 gWei
```

Comment:
If someone provides the `--gasPrice` parameter, this hardcoded value overrides that value... shouldn't it be the other way around? Seems like we might want to unify how the default and override behavior happens across the different networks.

Reply:
Hmm interesting, the problem is that the default gas price of 20 on ETH is too high for Polygon which can work with 1

Reply:
I'll remove this gas price for now, let's just let truffle auto detect it

Reply:
It seems like the gas price is still hardcoded for mainnet. IMO, we should make this hardcoded gas price just be the default and allow the --gasPrice to override it in the main config, but definitely open to other ideas/solutions you have!